postgresql_audit/base.py

```diff
@@ -55,32 +51,10 @@ def assign_actor(base, cls, actor_cls):
         cls.actor_id = sa.Column(sa.Text)
 
 
-def audit_table(table, exclude_columns=None):
-    args = [table.name]
-    if exclude_columns:
-        for column in exclude_columns:
-            if column not in table.c:
-                raise ImproperlyConfigured(
-                    "Could not configure versioning. Table '{}'' does "
-                    "not have a column named '{}'.".format(
-                        table.name, column
-                    )
-                )
-        args.append(array(exclude_columns))
-
-    query = sa.select(
-        [sa.func.audit.audit_table(*args)]
-    )
-    if query not in cached_statements:
-        cached_statements[query] = StatementExecutor(query)
-    listener = (table, 'after_create', cached_statements[query])
-    if not sa.event.contains(*listener):
-        sa.event.listen(*listener)
-
-
-def activity_base(base):
+def activity_base(base, schema='public'):
```

Comment:
Lets leave the default value to ``None`` and use whatever default value SQLAlchemy uses for the schema. Same applies to all functions that have this ``schema`` parameter.